src/main/java/org/folio/sidecar/integration/tm/TenantManagerClient.java

```diff
@@ -23,13 +23,19 @@
 
 @Log4j2
 @ApplicationScoped
-@RequiredArgsConstructor
 public class TenantManagerClient {
 
-  private final WebClient webClient;
+  private final WebClient webClientTls;
   private final JsonConverter jsonConverter;
   private final TenantManagerClientProperties clientProperties;
 
+  public TenantManagerClient(@Named("webClientTls") WebClient webClientTls, JsonConverter jsonConverter,
+                             TenantManagerClientProperties clientProperties) {
```

Comment:
you can add `@Named` annotation to the `lombok.config` and continue using `@RequiredArgsConstructor`